eclair-core/src/main/scala/fr/acinq/eclair/router/Router.scala

```diff
@@ -171,9 +171,18 @@ case object GetRoutingState
 
 case class RoutingState(channels: Iterable[PublicChannel], nodes: Iterable[NodeAnnouncement])
 
-case class Stash(updates: Map[ChannelUpdate, Set[ActorRef]], nodes: Map[NodeAnnouncement, Set[ActorRef]])
+// @formatter:off
+sealed trait GossipOrigin {
```

Comment:
I don't think we should make `actor` an attribute of `GossipOrigin`. Also, from what I gather the `RemoteGossip.actor` is a `Peer` whereas `LocalGossip.actor` is a `Channel`.